qaevents.md

```diff
@@ -0,0 +1,116 @@
+## Main Endpoint
+**GET /api/integration/qaevents**
+
+_Unsupported._ The suggestions can be retrieved only by source and target or via direct link, see the single entry and search method below. 
+
+## GET Single suggestion
+** GET /api/integration/qaevents/<:qaevent-id>
+
+Return a single suggestion:
+
+```json
+{
+  "id":"123e4567-e89b-12d3-a456-426614174000",
+  "originalId": "oai:www.openstarts.units.it:10077/21486",
+  "title":"Index nominum et rerum",
+  "trust":"0.375",
+  "eventDate": "2020/10/09 10:11 UTC",
+  "status": "PENDING",
+  "message" : {
+    "type":"doi",
+    "value":"10.18848/1447-9494/cgp/v15i09/45934",
+    "abstracts":"Abstract Complex functionality of the ...",
+    "acronym":"EPSILON",
+    "code":"277749",
+    "funder":"EC",
+    "fundingProgram":"FP7",
+    "jurisdiction":"EU",
+    "title":"Elliptic Pdes and Symmetry of Interfaces and Layers for Odd Nonlinearities"
+  },
+  type: "qaevent"
+}
+```
+
+Attributes
+* the *id* attribute is the event primary key
+* the *originalId* attribute is the oai identifier of the target publication
+* the *title* attribute is the title of the publication as provided by the correction's source
+* the *trust* attribute is the level of accuracy of the suggestion
```

Comment:
It looks like the value that comes back here is a fraction/percentage?  Can we describe the value better? E.g. is it a percentage of trust?  Does 1.0 mean 100% certain?  I think we should provide more info to people who use this endpoint